src/transaction/atomic_replication_helper.cpp

```diff
@@ -340,12 +345,31 @@ namespace cublog
 	//    be fixed, a client transactions manages to fix the page; IOW, how is the progress
 	//    of the "highest processed LSA" working wrt atomic replication sequences
 
+	if (prm_get_bool_value (PRM_ID_ER_LOG_PTS_ATOMIC_REPL_DEBUG))
+	  {
+	    constexpr int BUF_LEN_MAX = UCHAR_MAX;
+	    char buf[BUF_LEN_MAX];
+	    char *const buf_ptr = buf;
```

Comment:
Even without manipulating the pointer, `buf_ptr` is necessary instead of just using `buf`? (Same in others)

Reply:
fixed